bananas_api/tool_region/__main__.py

```diff
@@ -0,0 +1,113 @@
+"""
+Import an UN M49 CSV and ISO-3166-2 JSON file and generate regions.py.
+
+For UN M49 CSV, please see:
+  https://unstats.un.org/unsd/methodology/m49/overview
+
+For ISO-3166-2 JSON, please see:
+  Debian package iso-codes
+"""
+
+import click
+import datetime
+import json
+import os
+import unicodedata
+
+from openttd_helpers import click_helper
+
+
+@click_helper.command()
+@click.argument("m49_csv_file", type=click.Path(dir_okay=False, file_okay=True, exists=True))
+@click.argument("iso_3166_2_json_file", type=click.Path(dir_okay=False, file_okay=True, exists=True))
+def main(m49_csv_file, iso_3166_2_json_file):
+    regions = set()
+
+    with open(m49_csv_file) as fp:
+        for line in fp.readlines()[1:]:
```

Comment:
```suggestion
        next(fp)
        for line in fp:
```

? (possibly with a comment)